apache/superset
# fix: Bar chart order

### SUMMARY
The goal of this PR is to fix the order of the Bar chart

### BEFORE

https://user-images.githubusercontent.com/60598000/154296688-96327e55-73d1-42f6-a41e-7137fce73b8b.mp4

### AFTER

https://user-images.githubusercontent.com/60598000/154297218-6815a418-793c-49a2-b66b-08b6abaf981c.mp4

### TESTING INSTRUCTIONS
1. Open the sample Total Sales per Market (Grouped by Genre) Bar chart
2. Add a sort by metric
3. Make sure the order is correct

### ADDITIONAL INFORMATION
